src/system/PageLockModule/Api/LockingApi.php

```diff
@@ -24,6 +27,16 @@
 class LockingApi
 {
     /**
+     * Amount of required/opened accesses.
+     */
+    static $pageLockAccessCount = 0;
```

Comment:
to avoid that multiple counters could be instantiated (which may not happen anyway when using *only* container, but just to be sure)